src/partials/messenger.ts

```diff
@@ -79,6 +80,54 @@ class SendFileController extends DialogController {
     }
 }
 
+/**
+ * Handle settings
+ */
+class SettingsController{
+
+    public static $inject = ['$mdDialog','SettingsService','$window'];
+
+    public $mdDialog: ng.material.IDialogService;
+    public $window: ng.IWindowService
+    public settingsService: threema.SettingsService;
+    public activeElement: HTMLElement | null;
+
+    private theme: string;
+
+    constructor($mdDialog: ng.material.IDialogService,settingsService: threema.SettingsService,$window: ng.IWindowService) {
+        this.$mdDialog = $mdDialog;
+        this.$window = $window;
+        this.activeElement = document.activeElement as HTMLElement;
+        this.settingsService = settingsService;
+
+        this.theme = this.settingsService.getTheme();
+    }
+
+    public setTheme(name: string): void {
+        this.theme = name;
+        this.settingsService.setTheme(this.theme);
+    }
+
+    public cancel(): void {
+        this.$mdDialog.cancel();
+        this.done();
+    }
+
+    protected hide(data: any): void {
+        this.$mdDialog.hide(data);
+        this.done();
+    }
+
+    private done(): void {
+        if (this.activeElement !== null) {
+            // reset focus
+            this.$window.location.reload();
```

Comment:
I think so , there hase to be some way like the demo shows https://material.angularjs.org/1.1.0/demo/colors but i dont know how yet. I am looking more at applying theme colors consistent on all elements atm.